src/contracts/core/RewardsCoordinator.sol

```diff
@@ -477,8 +682,35 @@ contract RewardsCoordinator is
     }
 
     /// @inheritdoc IRewardsCoordinator
-    function operatorCommissionBips(address /* operator */, address /* avs */) external view returns (uint16) {
-        return globalOperatorCommissionBips;
+    function getOperatorAVSCommission(address operator, address avs) external view returns (uint16) {
+        OperatorCommission memory operatorCommission = operatorAVSCommissionBips[operator][avs];
+
+        if (operatorCommission.activatedAt == 0) {
+            // Return the Global Operator Commission if the operator commission has not been initialized.
+            return globalOperatorCommissionBips;
+        } else {
+            // Return the new commission if the new commission has been activated, else return the old commission.
+            return
+                (block.timestamp >= operatorCommission.activatedAt)
+                    ? operatorCommission.newCommissionBips
+                    : operatorCommission.oldCommissionBips;
+        }
+    }
+
+    /// @inheritdoc IRewardsCoordinator
+    function getOperatorPICommission(address operator) external view returns (uint16) {
+        OperatorCommission memory operatorCommission = operatorPICommissionBips[operator];
```

Comment:
We can dedupe the getter logic here too

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/837#discussion_r1819675600

Reply:
Thought about it a bit more and decided to refactor and dedupe it: [9f18c44](https://github.com/Layr-Labs/eigenlayer-contracts/pull/837/commits/9f18c445e548736086ffccd8df7ac57e25433ce5)